### ISSUE TITLE
Improve UI of toggle bar

### ISSUE THREAD

**Issue by Palakkgoyal** [narayan954/dummygram#778]
### Description

I think the toggle bar could look better. It can include some hover effects and active states.

### Steps to reproduce

Open the toggle bar by clicking on the navbar rightmost button

### Screenshots

![Screenshot 2023-07-13 165712](https://github.com/narayan954/dummygram/assets/116902573/c043e403-d4b8-4b09-b984-d264a44263d0)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I have checked the existing issues
- [X] I'm a GSSoC'23 contributor
- [X] I want to work on this issue

**Comment by narayan954** [comment]
Hi @Palakkgoyal actually am thinking of removing that whole component from navbar and implement it on the sidebar. Also this issue could be WIP as well. I'm thinking to remove the navbar completely and let there be a sidebar only. What do you think about that? Is current implementation better and should be enhanced or we should let that other implementation proceed? Later we may also a bar on right side if things workout with column and row mode both

**Comment by Palakkgoyal** [comment]
Yes, I think you are right but, there is a catch I tried dummygram yesterday on mobile and the sidebar came at the bottom of page and it looked to much cluttered with so many icons in it. I think there must be maximum 5 icons and we should move other to the profile page itself. Such as removing favorites from here and add it in the profile page as a toggle between posts and favorites.

**Comment by narayan954** [comment]
> Yes, I think you are right but, there is a catch I tried dummygram yesterday on mobile and the sidebar came at the bottom of page and it looked to much cluttered with so many icons in it. I think there must be maximum 5 icons and we should move other to the profile page itself. Such as removing favorites from here and add it in the profile page as a toggle between posts and favorites. What do you think?

yes that's right actually, I was also concerned about it, in desktop view it's okay, but won't be ok for mobile view. And to make it work on both, we can do as you're saying!

**Comment by narayan954** [comment]
See pr #803 it moves them into the profile page. 
Also I've a concern, as this pr will change the profile page,and you're working on profile page as well, which pr should go first?

**Comment by Palakkgoyal** [comment]
I think you should merge #803 and then, I will work on the UI.

**Comment by Palakkgoyal** [comment]
Hey, do you like the new hover effect on the sidebar? I think it looks nice in dark mode but, not in light mode. 
What do you think?

**Comment by narayan954** [comment]
> Hey, do you like the new hover effect on the sidebar? I think it looks nice in dark mode but, not in light mode. What do you think?

Nope, that's my doing. I just wanted to make it working... didn't check the colours and all much

**Comment by narayan954** [comment]
> I think you should merge #803 and then, I will work on the UI. After merging please let me know.

aight, should do by eod
